lib/graphql_schema.rb

```diff
@@ -191,7 +201,7 @@ def fields(include_deprecated: false)
     end
 
     def input_fields
-      @input_fields ||= @hash.fetch('inputFields').map{ |field_hash| Field.new(field_hash) }
+      @input_fields ||= @hash.fetch('inputFields').map{ |field_hash| InputField.new(field_hash) }
```

Comment:
To match the GraphQL introspection schema we should rename the `Argument` class to `InputValue` and use it here.  For instance, it doesn't make sense for an InputField class to have an `args` method.